CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# How to Contribute #
+
+We're always looking for people to help make SbManager even better, there are a number of ways to contribute.
+
+## Documentation ##
+Setup guides, FAQ, the more information we have the better.
+
+## Development ##
+
+### Tools required ###
+- Visual Studio 2015
+- HTML/Javascript editor of choice (Sublime Text/Webstorm/Atom/Visual Studio/etc)
+- git
+
+### Getting started ###
+
+1.  Fork SbManager
+2.  Clone (master branch)
+3.  Compile in Visual Studio
+
+### Contributing Code ###
+- If you're adding a new, already requested feature, please comment on [Github Issues](https://github.com/GlobalX/SbManager/issues "Github Issues") so work is not duplicated (If you want to add something not already on there, please talk to us first)
+- Rebase from SbManager's develop branch, don't merge
+- Make meaningful commits, or squash them
+- Feel free to make a pull request before work is complete, this will let us see where its at and make comments/suggest improvements
+- Reach out to us on [Gitter](https://gitter.im/GlobalX/SbManager) if you have any questions
+- Add unit tests
+- One feature/bug fix per pull request to keep things clean and easy to understand
+- Use 4 spaces instead of tabs
```

Comment:
Visual Studio doesn't have editor config support yet (slated for 2017). I'll leave this here but add a editorconfig file as well.